server.js

```diff
@@ -27,21 +27,18 @@ app.prepare().then(() => {
     ) {
       // Enforce https protocol and remove www from host/
       res.writeHead(301, {
-        Location:
-          'https://' + req.headers['host'].replace(/^www\./, '') + req.url
+        Location: 'https://' + req.headers.host.replace(/^www\./, '') + req.url
       })
       res.end()
     } else if (req.headers.host === 'man.dvc.org') {
       // man.dvc.org/{cmd} -> dvc.org/doc/command-reference/{cmd},
       // replace - for / in {cmd} except for /get-url, /import-url
-      let new_path
-      if (['/get-url', '/import-url'].indexOf(pathname) < 0) {
-        new_path = pathname.replace('-', '/')
-      } else {
-        new_path = pathname
-      }
       res.writeHead(301, {
-        Location: 'https://dvc.org/doc/command-reference' + new_path
+        Location:
+          'https://dvc.org/doc/command-reference' +
+          (['/get-url', '/import-url'].indexOf(pathname) < 0
+            ? pathname.replace('-', '/')
+            : pathname)
       })
```

Comment:
This is not possible to test in the review app... However I tried it locally duplicating this else case and substituting `dvc.org` with `localhost:3000` and the redirects work correctly: from man.localhost:3000/get-url and /import-url to localhost:3000/doc/command-reference/get-url and /import-url respectively, but from any other man.localhost:3000 path with dashes (such as /cache-dir) to localhost:3000/doc/command-reference{path with slashes} (e.g. to .../command-reference/cache/dir).

Please try to test this locally before merging to see if this somehow fixes the redirect on prod.